docs/developer-docs/latest/setup-deployment-guides/installation/cli.md

```diff
@@ -38,84 +68,110 @@ Strapi currently support the following databases.
 | MariaDB    | 10.1            |
 | MongoDB    | 3.6             |
```

Comment:
Following the last title, "Strapi currently supports the following databases". Wouldn't it be technically correct to include SQLite in this list? Since it's referenced below as the Database of choice when the `--quickstart` flag is applied

Reply:
:+1:

Reply:
Maybe I missed something, Richard, but SQLite already is in the list of supported databases:

![Screenshot 2021-06-21 at 15 53 24](https://user-images.githubusercontent.com/4233866/122773879-03345780-d2a9-11eb-85e5-32c36a1ee173.png)

What we did on this part was just added the note about dropping MongoDB.

Reply:
I believe I was looking at a clipped view of the table. My bad. Looks good to me.